src/figuresClasses.ts

```diff
@@ -1,19 +1,71 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
```

Comment:
create a literal type for shapes. it should be some exact value (`triangle' | 'circle' | 'rectangle'`), not any string

Reply:
the same with color